infra/scripts/test-end-to-end.sh

```diff
@@ -86,8 +76,30 @@ feast:
       subscriptions:
         - name: "*"
           project: "*"
+  core-authentication:
+    enabled: $ENABLE_AUTH 
+    provider: google 
+  security:
+    authentication:
+      enabled: $ENABLE_AUTH
+      provider: jwt
+    authorization:
+      enabled: false
+      provider: none
 EOF
 
+if [[ ${ENABLE_AUTH} = "true" ]]; 
+  then
+    print_banner "Starting 'Feast core with auth'."
```

Comment:
Can we remote the nested quotes here and the full stop.

Reply:
remove quotes and period